externals/kyuubi-hive-sql-engine/src/test/scala/org/apache/kyuubi/engine/hive/operation/HiveOperationSuite.scala

```diff
@@ -31,6 +31,13 @@ class HiveOperationSuite extends HiveJDBCTestHelper {
     "jdbc:hive2://" + HiveSQLEngine.currentEngine.get.frontendServices.head.connectionUrl + "/;"
   }
 
+  test("get catalogs") {
+    withJdbcStatement() { statement =>
+      val catalogs = statement.getConnection.getMetaData.getCatalogs
+      assert(!catalogs.next())
```

Comment:
can we also check the value?

Reply:
This line checks whether the `catalogs` is empty list.